packages/application/src/domain/application/application.ts

```diff
@@ -529,51 +548,49 @@ rejectGovernanceReview(details: GovernanceReviewRejectedData) {
 
   applyKYCApproved(event: KYCApproved) {
     console.log(' applyKYCApproved')
-    if (this.applicationStatus===ApplicationStatus.KYC_PHASE) {
-      this.status["KYC Submitted"] ??= event.timestamp.getTime()
+    if (this.applicationStatus === ApplicationStatus.KYC_PHASE) {
+      this.status['KYC Submitted'] ??= event.timestamp.getTime()
       this.applicationStatus = ApplicationStatus.GOVERNANCE_REVIEW_PHASE
     }
   }
-//check that it should be KYCApproved. 
+  //check that it should be KYCApproved.
   applyKYCRevoked(_: KYCApproved) {
     this.applicationStatus = ApplicationStatus.GOVERNANCE_REVIEW_PHASE
-    this.status["KYC Submitted"] = null
+    this.status['KYC Submitted'] = null
   }
 
   applyKYCRejected(event: KYCRejected) {
-    this.status["KYC Failed"] = event.timestamp.getTime()
+    this.status['KYC Failed'] = event.timestamp.getTime()
   }
 
   applyGovernanceReviewStarted(event: GovernanceReviewStarted) {
     console.log('applyGovernanceReviewStarted')
     if (this.applicationStatus === ApplicationStatus.IN_REFRESH) {
-      this.status["In Refresh"] ??= event.timestamp.getTime()
+      this.status['In Refresh'] ??= event.timestamp.getTime()
       this.applicationStatus = ApplicationStatus.GOVERNANCE_REVIEW_PHASE
     }
   }
 
   applyGovernanceReviewApproved(event: GovernanceReviewApproved) {
     console.log('applyGovernanceReviewApproved')
-     if (this.applicationStatus === ApplicationStatus.GOVERNANCE_REVIEW_PHASE) {
-      this.status["Approved"] ??= event.timestamp.getTime()
+    if (this.applicationStatus === ApplicationStatus.GOVERNANCE_REVIEW_PHASE) {
+      this.status['Approved'] ??= event.timestamp.getTime()
     }
     //this.applicationStatus = ApplicationStatus.APPROVED
     this.applicationInstructions = event.applicationInstructions
   }
 
   applyGovernanceReviewRejected(event: GovernanceReviewRejected) {
     this.applicationStatus = ApplicationStatus.REJECTED
-    if (!this.status["Declined"]) {
-      this.status["Declined"]=event.timestamp.getTime()
+    if (!this.status['Declined']) {
+      this.status['Declined'] = event.timestamp.getTime()
     }
   }
 
   applyRKHApprovalStarted(event: RKHApprovalStarted) {
     console.log('applyRKHApprovalStarted')
     this.applicationStatus = ApplicationStatus.RKH_APPROVAL_PHASE
-    this.allocationTooling = [],
-    this.pathway = 'RKH',
-    this.ma_address = 'f080'
+    ;(this.allocationTooling = []), (this.pathway = 'RKH'), (this.ma_address = 'f080')
```

Comment:
I think these commas were a bug in the original code. There's no need for a tail return or anything here.

```suggestion
    this.allocationTooling = []
    this.pathway = 'RKH'
    this.ma_address = 'f080'
```